docs/source/getting_started/content/proc_configuring-the-virtual-machine.adoc

```diff
@@ -0,0 +1,52 @@
+[id="configuring-the-virtual-machine_{context}"]
+= Configuring the virtual machine
+
+Use the `cpus` and `memory` properties to configure the default number of vCPUs and amount of memory available to the {prod} virtual machine, respectively.
```

Comment:
This is fine but getting back to your comment about minimalism, I would have written this:

> The `cpus` property configures the default number of vCPUs.
> The  `memory` property configures the amount of memory available to the {prod} virtual machine.

Reply:
Fair enough, but those sentences seem more like part of the procedure itself rather than an introduction to the steps being performed in my opinion. :)